Bottom: e72a3635922f21ffe2f69bb8f6d0ceef979966e2
Top:    0f21cceb4d96e4896342947541fd9c5faedbddd2
Author: Ian Kent <raven@themaw.net>
Date:   2013-05-11 10:12:11 +0800

Refresh of autofs-5.0.7-make-dump-maps-check-for-duplicate-indirect-mounts.patch

---

diff --git a/lib/master.c b/lib/master.c
index 36d677c..97c849f 100644
--- a/lib/master.c
+++ b/lib/master.c
@@ -1365,8 +1365,8 @@ int master_show_mounts(struct master *master)
 					i = 1;
 				}
 				if (count && ap->type == LKP_INDIRECT)
-					printf("  duplicate map source will be"
-					       " ignored at run time\n");
+					printf("  duplicate indirect map source"
+					       " will be ignored at run time\n");
 				if (source->argc > 1) {
 					printf("  arguments: ");
 					for (; i < source->argc; i++)
